Formula/chrpath.rb

```diff
@@ -0,0 +1,19 @@
+class Chrpath < Formula
+  desc "Tool to edit the rpath in ELF binaries"
+  homepage "https://tracker.debian.org/pkg/chrpath"
+  url "https://deb.debian.org/debian/pool/main/c/chrpath/chrpath_0.16.orig.tar.gz"
+  sha256 "bb0d4c54bac2990e1bdf8132f2c9477ae752859d523e141e72b3b11a12c26e7b"
+  license "GPL-2.0-or-later"
+
+  depends_on :linux
+
+  def install
+    system "./configure", *std_configure_args, "--disable-silent-rules"
+    # SUBDIRS= skips building a test which is not used (and fails on macOS):
+    system "make", "install", "SUBDIRS="
+  end
+
+  test do
+    assert_match "chrpath version #{version}", shell_output("#{bin}/chrpath -v")
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
Thanks, added several test cases testing the functionality now.